packages/react-toolbar/Spec.md

```diff
@@ -2,23 +2,26 @@
 
 ## Background
 
-_Description and use cases of this component_
+### Definition
```

Comment:
I would suggest creating a separate PR for the spec so there can be more feedback for it during the development time